### PR TITLE
Update mdn_plus_privacy.md

### PR DESCRIPTION
**Pull request by jejohnsonmoz**
Update for Firefox > Mozilla Account change + minor changes in last paragraph to align with other recently updated Notices.

### CODE REVIEW

File: en/mdn_plus_privacy.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## <span class="privacy-header-firefox">MDN Plus</span> <span class="privacy-header-policy">Privacy Notice</span>
 
-Version 1.1, Effective May 24, 2023
+Version 1.2, Effective November 1, 2023
 {: datetime="2023-05-24" }
```
Comment (peiying2) [inline en/mdn_plus_privacy.md:4]: change the date here too.

File: en/mdn_plus_privacy.md [hunk 2]
```diff
@@ -31,10 +31,9 @@ MDN Plus doesn’t share any of your data with third parties.
 
 ### Data privacy questions? {: #markup }
 
-Much of the information that we store about our Firefox Account users is easily accessible by logging into your account. You can learn more about  managing your data [here](https://support.mozilla.org/products/privacy-and-security/user-control). To make requests, please contact us through our [Data Subject Access Request Portal](https://privacyportal.onetrust.com/webform/1350748f-7139-405c-8188-22740b3b5587/4ba08202-2ede-4934-a89e-f0b0870f95f0).
+Much of the information that we store about our Firefox Mozilla Account users is easily accessible by signing in to your account, where you can also update your data sharing settings. To make requests regarding your personal data, please contact us through our [Data Subject Access Request Portal](https://privacyportal.onetrust.com/webform/1350748f-7139-405c-8188-22740b3b5587/4ba08202-2ede-4934-a89e-f0b0870f95f0).
```
Comment (peiying2) [inline en/mdn_plus_privacy.md:34]: Remove Firefox

File: en/mdn_plus_privacy.md [hunk 3]
```diff
@@ -13,15 +13,15 @@ This privacy notice explains what data MDN Plus collects, shares, and why. We al
 
 ### Information we collect {: #InfoWeCollect }
 
-__Firefox Account information.__ This service requires a Firefox Account, which provides MDN Plus read-only access to your Firefox Account profile information, for example your user ID, display name, avatar, email address, locale, and active product subscriptions. Learn more about [Firefox Account data practices](https://www.mozilla.org/privacy/firefox/#firefox-accounts-join-firefox).
+__Mozilla Account information.__ This service requires a Mozilla Account, which provides MDN Plus read-only access to your Mozilla Account profile information, for example your user ID, display name, avatar, email address, locale, and active product subscriptions. Learn more about [Mozilla Account data practices](https://www.mozilla.org/privacy/firefox/#firefox-accounts-join-firefox).
```
Comment (peiying2) [inline en/mdn_plus_privacy.md:16]: Should be lower case `a` in `Mozilla account`. We used to call it Firefox Account but later it changed to the lower `a`.  Make sure to update the entire document accordingly.

File: en/mdn_plus_privacy.md [hunk 4]
```diff
@@ -13,15 +13,15 @@ This privacy notice explains what data MDN Plus collects, shares, and why. We al
...
 
 __MDN Plus Account Information.__ When you use MDN Plus features, we receive the data required to provide those features to you. For instance, if you set saved pages, we receive your saved pages.
```
Comment (peiying2) [inline en/mdn_plus_privacy.md:18]: lower case A and I, so the style is consistent with other bullet points: `MDN Plus account information`

File: en/mdn_plus_privacy.md [hunk 5]
```diff
@@ -31,10 +31,9 @@ MDN Plus doesn’t share any of your data with third parties.
...
+Much of the information that we store about our Firefox Mozilla Account users is easily accessible by signing in to your account, where you can also update your data sharing settings. To make requests regarding your personal data, please contact us through our [Data Subject Access Request Portal](https://privacyportal.onetrust.com/webform/1350748f-7139-405c-8188-22740b3b5587/4ba08202-2ede-4934-a89e-f0b0870f95f0).
 
 If you have any other questions regarding our privacy practices, please contact us at compliance@mozilla.com.
-
-We respond to all requests we receive from individuals wishing to exercise their data protection rights in accordance with applicable data protection laws.
+We respond to all requests we receive from individuals wishing to exercise their data protection rights regardless of where the individual lives. We will honor your request unless a legal requirement prevents us from doing so or a legal exception applies.
```
Comment (peiying2) [inline en/mdn_plus_privacy.md:37]: If this is a separate paragraph, add a line space. Otherwise, wrap it to where the email address ends.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Global change is needed from Mozilla Account to Mozilla account. And some other comments to address.

File: en/mdn_plus_privacy.md (hunk 1 above)
Comment (jejohnsonmoz) [reply en/mdn_plus_privacy.md:4]: Done

File: en/mdn_plus_privacy.md (hunk 3 above)
Comment (jejohnsonmoz) [reply en/mdn_plus_privacy.md:16]: Sorry! Done (I think)

File: en/mdn_plus_privacy.md (hunk 5 above)
Comment (jejohnsonmoz) [reply en/mdn_plus_privacy.md:37]: Done

File: en/mdn_plus_privacy.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ## <span class="privacy-header-firefox">MDN Plus</span> <span class="privacy-header-policy">Privacy Notice</span>
 
-Version 1.1, Effective May 24, 2023
-{: datetime="2023-05-24" }
+Version 1.2, Effective November 1, 2023
+{: datetime="2023-11-1" }
```
Comment (flodolo) [inline en/mdn_plus_privacy.md:4]:
```suggestion
{: datetime="2023-11-01" }
```

**Review by peiying2 — APPROVED** [review]

**Pull request merged**